examples/calculations/submit_using_config.py

```diff
@@ -2,32 +2,45 @@
 
 from __future__ import annotations
 
+from pathlib import Path
+
 from aiida.engine import run_get_node
 from aiida.orm import load_code
 from aiida.plugins import CalculationFactory
 
+import aiida_mlip
 from aiida_mlip.data.config import JanusConfigfile
-from aiida_mlip.helpers.help_load import load_structure
+from aiida_mlip.helpers.help_load import load_model, load_structure
+
+# __file__ changes depending on where verdi run is called
+DATA_PATH = Path(aiida_mlip.__file__).parent.parent / "tests" / "calculations"
```

Comment:
Is this relevant/cleaner? https://docs.python.org/3/library/pkgutil.html#pkgutil.get_data

Reply:
It looks like it's useful for loading the contents of a file quite nicely, but I don't think it gives us a nicer way to get the path, which we need to construct the AiiDA data types?

Reply:
True, the data aren't in the package.